routers/repo/issue.go

```diff
@@ -351,6 +351,7 @@ func NewIssue(ctx *context.Context) {
 	ctx.Data["PageIsIssueList"] = true
 	ctx.Data["RequireHighlightJS"] = true
 	ctx.Data["RequireSimpleMDE"] = true
+	ctx.Data["RequireTribute"] = true
```

Comment:
I believe we also should require tribute in [CompareAndPullRequest](https://github.com/go-gitea/gitea/blob/0d4c63fd70d0b255a9a17352cc47ec2b6ca13440/routers/repo/pull.go#L699)